createcert.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+
+function createCert(){
+	if [[ ! -f $HOME/.config/nuntius/nuntius.pem || ! -f $HOME/.config/nuntius/nuntius.key ]]; then
+		echo "Creating cert..."
+		cd $HOME/.config/nuntius
+		openssl genrsa -out nuntius.key 2048
```

Comment:
I know the script is a temporary solution and does not need to be very robust, but let's at least check that openssl is available